.github/scripts/svgo.config.js

```diff
@@ -0,0 +1,53 @@
+module.exports = {
+  plugins: [
+    {
+      name: 'preset-default'
+    },
+    {
+      name: 'removeDimensions',
+      params: {
+        overrides: {
+          removeDimensions: true,
+        }
+      }
+    },
+    {
+      name: 'convertStyleToAttrs',
+      params: {
+        overrides: {
+          convertStyleToAttrs: true,
+        }
+      }
+    },
+    {
+      name: 'removeRasterImages',
+      params: {
+        overrides: {
+          removeRasterImages: true,
```

Comment:
I think we want to just let the user know that rasterImages can't be used instead of removing them. 

At least we shouldn't remove them automatically imo. It makes it harder to review, since we have to check the previeous commit to know if parts of the logo got removed. If say the shadow was a raster image, and it was removed, then the icon has been modified too much to be usable as an `original` version.

For these reasons think it would be better to just add a raster image check to the `check-bot`, so that we can't merge PRs with raster images in the icons

Reply:
I'm ok with it. Removing this plugin in next commit! 🙂
> *That other bot to check raster images, should be easy to do, but if it's in Python I rather leave it to other person who knows better how to work with that language.* 😀 *Still, I can help with the Regular Expressions.* 😉